appveyor.yml

```diff
@@ -17,8 +15,9 @@ build_script:
 
 test:
   assemblies:
-    - Cli-CredentialHelper.Test\bin\$(configuration)\Microsoft.Alm.*.Test.dll
-    - Microsoft.Alm.*.Test\bin\$(configuration)\Microsoft.Alm.*.Test.dll
+    - Cli-CredentialHelper.Test\bin\$(configuration)\*.Test.dll
+    - GitHub.Authentication.Test\bin\$(configuration)\*.Test.dll
+    - Microsoft.Alm.*.Test\bin\$(configuration)\*.Test.dll
```

Comment:
`*.Test\bin\$(configuration)\*.Test.dll` might work too. I will try that since it's simpler.